src/applications/simple-forms/21-0972/definitions/constants.js

```diff
@@ -6,10 +6,10 @@ export const claimantIdentificationKeys = [
 ];
 
 export const claimantIdentificationOptions = {
```

Comment:
Better to use Object.freeze({...}) to truly prevent the _props_ from being inadvertantly overwritten in your app-code.  [i'm circling back to my older forms to do the same.]